app/services/platform-apps/index.ts

```diff
@@ -134,6 +135,7 @@ export class PlatformAppsService extends
   appUnload = new Subject<string>();
 
   private localStorageKey = 'PlatformAppsUnpacked';
+  private baseCdnUrl = 'https://s3.amazonaws.com/streamlabs-platform';
```

Comment:
We don't want to hard code this.  We want to add `appUrl` which you added last week as part of your production app stuff: https://github.com/stream-labs/streamlabs-obs/pull/840/files#diff-5ff6bf525e7ffbac00adf915bb8e26f2R99

Essentially here's the logic:
- Is this app running unpacked?  Do nothing past what we're normally doing
- Is this app running in production?  Make sure any script being loaded has the same beginning of the URL as `appUrl` we got back from the server.  That ensures the script belongs to this app.